packages/kit/scripts/extract-types.js

```diff
@@ -117,9 +131,9 @@ modules.push({
 
 			modules.push({
 				name,
-				comment,
+				comment: comment.replace(/https:\/\/kit\.svelte\.dev/g, ''),
```

Comment:
this could use a comment. I don't understand why `kit.svelte.dev` was prepended to all the links and then is being removed here